worlds/tunic/__init__.py

```diff
@@ -94,19 +97,24 @@ def generate_early(self) -> None:
         # Universal tracker stuff, shouldn't do anything in standard gen
         if hasattr(self.multiworld, "re_gen_passthrough"):
```

Comment:
This is probably out of scope, but you could combine the two `if` clauses into one with an `and`

Reply:
Yeahh, pycharm doesn't like that, unfortunately. Gives me better peace of mind not to have pycharm say "oh no but wait maybe re_gen_passthrough isn't in self.multiworld???" on the second half of the line

Reply:
Oh that problem, fun. Yeah 👍